docker/py39/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9
+
+RUN apt-get update \
+    && apt-get install -y \
+        hdf5-tools \
+        curl
+
+RUN curl https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+
+RUN apt-get install git-lfs
```

Comment:
These `RUN` commands should also be done together with `&&` to prevent caching the `git-lfs` curl downloaded files in an intermediate Docker image layer (which bloats the final image size)